modules/sfm/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+
+if (NOT HAVE_EIGEN)
+  message(STATUS "Module opencv_sfm disabled because Eigen library is required.")
+  ocv_module_disable(sfm)
+  return()
+endif()
+
+set(the_description "SFM algorithms")
+
+ocv_warnings_disable(CMAKE_CXX_FLAGS -Wundef -Winvalid-pch)
+
+if(UNIX)
+  if(CMAKE_COMPILER_IS_GNUCXX OR CV_ICC)
+     set(CMAKE_CXX_FLAGS "${CMAKE_CSS_FLAGS} -fPIC")
+  endif()
+endif()
+
+
+### LIBMV LIGHT DEFINITIONS ###
+
+set(LIBMV_LIGHT_INCLUDES
+  src/libmv_light
+  src/libmv_light/third_party/ssba
+  "${OpenCV_SOURCE_DIR}/include/opencv"
+)
+
+set(LIBMV_LIGHT_LIBS
+  reconstruction
+  correspondence
+  camera
+  multiview
+  multiview_test_data
+  numeric
+)
+
+
+### LIBMV LIGHT EXTERNAL DEPENDENCIES ###
+
+find_package( Ceres QUIET )
+
+if ( Ceres_FOUND )
+  add_definitions("-DCERES_FOUND=1")
+  list(APPEND LIBMV_LIGHT_LIBS simple_pipeline)
+else ()
+  add_definitions("-DCERES_FOUND=0")
+  message(STATUS "CERES support is disabled. Ceres Solver for reconstruction API is required.")
+endif ()
+
+
+### DEFINE OPENCV SFM MODULE DEPENDENCIES ###
+
+set(OPENCV_SFM_DEPS
+  opencv_core
+  opencv_calib3d
+  opencv_features2d
+  opencv_xfeatures2d
+)
+
+
+### CREATE OPENCV SFM MODULE ###
```

Comment:
Problem is here: http://pullrequest.opencv.org/buildbot/builders/precommit-contrib_linux64/builds/1303/steps/compile%20release/logs/stdio

P.S. I updated code block in comment above with compiler flags (need to remove ocv_warnings_disable and fPIC from the beginning of file).